wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/functions.php

```diff
@@ -203,3 +230,104 @@ function wporg_ov_customizer( $wp_customize ) {
 	) );
 }
 add_action( 'customize_register', __NAMESPACE__ . '\wporg_ov_customizer' );
+
+/*
+	=====================================
+	Openverse standalone site redirection
+	=====================================
+ */
+
+/**
+ * This is the URL at which the standalone Openverse site is hosted. When
+ * redirect is enabled (see setting `ov_is_redirect_enabled`), the theme
+ * redirects all incoming requests to the right URL on this domain.
+ *
+ * Note: Do not put a trailing slash '/' in this URL, as it will cause problems.
+ */
+if ( !defined( 'OPENVERSE_STANDALONE_URL' ) ) {
+	define( 'OPENVERSE_STANDALONE_URL', 'https://openverse.wordpress.net' );
+}
```

Comment:
I wonder if this variable should be a theme variable instead? That way if we ever moved the location (say to `openverse.org`) we wouldn't need to redeploy the theme, just update the variable.

Edit: I see you actually already added it as a theme variable, but I don't see how it is used. Is there some magic going on that wires up the constant to the theme variable when the theme variable is defined?

Reply:
There's no magic, it's explicitly set as the default value [here](https://github.com/WordPress/wordpress.org/pull/98#pullrequestreview-1146887519).

Reply:
Right... but where is the theme variable used? I only see the constant being used in the code, not the theme variable. That's where my confusion is coming from.

Reply:
Good catch @sarayourfriend, thank you! Now fixed that in 0083dd5.